### ISSUE TITLE
`attributes` rule gives false error in protocol implementation after @testable

### ISSUE THREAD

**Issue by ldiqual** [realm/SwiftLint#2125]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Bug Report

`attributes` rules gives a false error on Xcode 9.3: 

```
File.swift:15:15: warning: Attributes Violation: Attributes should be on their own lines in functions and types, but on the same line as variables and imports. (attributes)
```

Code:

```
import SomeFramework

@testable import SomeApp

private final class SomeClass: SomeProtocol { // error here
    ...
}
```

### Environment

* SwiftLint version: 0.25.0
* Installation method used: cocoapods
* Paste your configuration file:

```yml
attributes:
  always_on_line_above:
    - "@objc"
    - "@nonobjc"
    - "@available"
    - "@discardableResult"
  always_on_same_line:
    - "@IBOutlet"
    - "@IBAction"
    - "@NSManaged"
    - "@testable"
```

* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)? Yes, one at the root folder and one in `AppTests`.

* Which Xcode version are you using (check `xcode-select -p`)? Xcode 9.3 9E145
* Do you have a sample that shows the issue? Yes, see above

**Comment by marcelofabri** [comment]
@ldiqual Was this introduced on Xcode 9.3/Swift 4.1?

**Comment by keith** [comment]
Another repro case:

```swift
@testable import foo

private let bar = ""
```

I can confirm this only happens for this case with Xcode 9.3/Swift 4.1.
